src/main.cpp

```diff
@@ -68,6 +68,7 @@ void print_usage_and_exit (std::string error_msg = "") {
     std::cout << "\t-d, --out-dir=<out-dir>   Output directory\n";
     std::cout << "\t-s, --seed=<seed>         Predefined seed (it is accepted in form of SSS or VV_SSS)\n";
     std::cout << "\t-m, --bit-mode=<32/64>    Generated test's bit mode\n";
+    std::cout << "\t--windows=<true/false>    Enable windows-compatible mode\n";
```

Comment:
Why adding it as a command line option, I don't think that we care about cross compilation. And we can detect Windows host when compiling on Windows, right?

Reply:
Fixed